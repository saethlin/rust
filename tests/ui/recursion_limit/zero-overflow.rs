//~ ERROR overflow evaluating the requirement `{closure@rt::lang_start<()>::{closure#0}}: Freeze`
//~| HELP consider increasing the recursion limit
//@ build-fail

#![recursion_limit = "0"]

fn main() {}
