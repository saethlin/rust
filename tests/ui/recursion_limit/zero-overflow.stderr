error[E0275]: overflow evaluating the requirement `{closure@rt::lang_start<()>::{closure#0}}: Freeze`
   |
   = help: consider increasing the recursion limit by adding a `#![recursion_limit = "2"]` attribute to your crate (`zero_overflow`)

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0275`.
